src/feedlistformaction.cpp

```diff
@@ -127,11 +127,11 @@ bool FeedListFormAction::process_operation(Operation op,
 		// i18n: This string is related to the letters in parentheses in the
 		// "Sort by (f)irsttag/..." and "Reverse Sort by
 		// (f)irsttag/..." messages
-		std::string input_options = _("ftauln");
+		std::string input_options = _("ftaulsn");
```

Comment:
I've been a bit too slow with reviewing.
We are currently preparing for a release.
One of the steps we take is freezing the translatable strings so I will wait with merging this PR until after the release (planned somewhere around March 23rd)